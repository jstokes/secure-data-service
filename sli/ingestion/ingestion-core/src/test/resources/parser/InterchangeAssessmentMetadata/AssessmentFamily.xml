<?xml version="1.0" encoding="UTF-8"?>
<InterchangeAssessmentMetadata xmlns:sli="http://slc-sli/ed-org/0.1"
 xmlns="http://ed-fi.org/0100"
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 xsi:schemaLocation="http://ed-fi.org/0100 file:/C:/beta/sli/sli/edfi-schema/src/main/resources/edfiXsd-SLI/SLI-Interchange-AssessmentMetadata.xsd">
    <AssessmentFamily id="ID001">
        <AssessmentFamilyTitle>HOjhWE6_IG1Anxl5</AssessmentFamilyTitle>
        <AssessmentFamilyIdentificationCode IdentificationSystem="School" AssigningOrganizationCode="WLQ">
            <ID>Jr</ID>
        </AssessmentFamilyIdentificationCode>
        <AssessmentCategory>Mental ability (intelligence) test</AssessmentCategory>
        <AcademicSubject>Composite</AcademicSubject>
        <GradeLevelAssessed>Grade 13</GradeLevelAssessed>
        <LowestGradeLevelAssessed>Tenth grade</LowestGradeLevelAssessed>
        <ContentStandard>Advanced Placement</ContentStandard>
        <Version>-1164435820</Version>
        <RevisionDate>2014-05-09</RevisionDate>
        <Nomenclature>Kfr87Oas</Nomenclature>
        <AssessmentPeriods id="ID003" ref="ID003">
            <CodeValue>WPhxGcxrV</CodeValue>
        </AssessmentPeriods>
        <AssessmentFamilyReference id="ID005" ref="ID002">
            <AssessmentFamilyIdentity>
                <AssessmentFamilyIdentificationCode IdentificationSystem="District" AssigningOrganizationCode="u.zXl85h0sKc9w9DUEiQLUX1s5H8Bs">
                    <ID>kkG3hnH</ID>
                </AssessmentFamilyIdentificationCode>
            </AssessmentFamilyIdentity>
        </AssessmentFamilyReference>
    </AssessmentFamily>
</InterchangeAssessmentMetadata>
