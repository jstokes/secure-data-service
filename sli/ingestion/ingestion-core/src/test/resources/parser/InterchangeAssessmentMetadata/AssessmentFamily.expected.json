{
  "a_id":"ID001",
  "AssessmentFamilyTitle":{"_value":"HOjhWE6_IG1Anxl5"},
  "AssessmentFamilyIdentificationCode":[{
    "a_IdentificationSystem":"School",
    "a_AssigningOrganizationCode":"WLQ",
    "ID":{"_value":"Jr"}
  }],
  "AssessmentCategory":{"_value":"Mental ability (intelligence) test"},
  "AcademicSubject":{"_value":"Composite"},
  "GradeLevelAssessed":{"_value":"Grade 13"},
  "LowestGradeLevelAssessed":{"_value":"Tenth grade"},
  "ContentStandard":{"_value":"Advanced Placement"},
  "Version":-1164435820,
  "RevisionDate":{"_value":"2014-05-09"},
  "Nomenclature":{"_value":"Kfr87Oas"},
  "AssessmentPeriods":[{
    "a_id":"ID003",
    "a_ref":"ID003",
    "CodeValue":{"_value":"WPhxGcxrV"}
  }],
  "AssessmentFamilyReference":{
    "a_id":"ID005",
    "a_ref":"ID002",
    "AssessmentFamilyIdentity":{
      "AssessmentFamilyIdentificationCode":[{
        "a_IdentificationSystem":"District",
        "a_AssigningOrganizationCode":"u.zXl85h0sKc9w9DUEiQLUX1s5H8Bs",
        "ID":{"_value":"kkG3hnH"}
      }]
    }
  }
}